skills/ramenctl-test-run/SKILL.md

```
@@ -0,0 +1,117 @@
+
+The test runs through the full DR flow for each test case:
+deploy, protect, failover, relocate, unprotect, undeploy.
+
+**This typically takes 15-20 minutes per test case.** Multiple test cases
+run in parallel.
+
+### Step 3: Check the result
+
+**On success:**
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)